telco-core/configuration/core-baseline.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: PolicyGenerator
+metadata:
+  name: core-baseline-16
+policyDefaults:
+  namespace: ztp-core-policies
+  policySets: []
+  placement:
+    clusterSelectors:
+      common: "core"
+      version: "4.16"
+  remediationAction: "inform"
+  #policyAnnotations:
+  #  ran.openshift.io/ztp-deploy-wave: "1"
+policies:
+  # Base cluster configuration
+  - name: core-cluster-config-4.16
+    policyAnnotations:
+      ran.openshift.io/ztp-deploy-wave: "1"
+    manifests:
+      - path: reference-crs/required/other/operator-hub.yaml
+      - path: reference-crs/required/other/catalog-source.yaml
+        patches:
+        - spec:
+            image: registry.redhat.io/redhat/redhat-operator-index:v4.16
```

Comment:
@imiller0 wdyt about moving version specific policies to a dedicated file so later they  could be easily unbounded during cluster update(when catsrc or subscription channel changes)?

Reply:
My thinking is that there would be one copy of each of these policies per version. So there would be a core-baseline-4.16, core-overlay-4.16, core-baseline-4.18, core-overlay-4.18, .... (same for any customer specific policy). That allows for there to be an update made in any reference configuration file that would automatically get updated for the user. WDYT?

Reply:
OK, that makes sense, thanks